### ISSUE TITLE
chrome.sockets.tcp on iPad: can't disconnect

### ISSUE THREAD

**Issue by tucano** [MobileChromeApps/cordova-plugin-chrome-apps-sockets-tcp#20]
Hello, I am developing a **cordova/cca** application with **cordova-plugin-chrome-apps-sockets-tcp 1.3.6.**

Everything fine in chrome (as extension) and in android, but in iPad (iOs 9.3.5) I can't disconnect the sockets.

If I try (even from command line using Safari Develop):

```
chrome.sockets.tcp.getSockets( function(s){ console.log(JSON.stringify(s))} )
```

I get:

```
[
{"socketId":1,"name":"","bufferSize":4096,"connected":false,"persistent":false,"paused":false},    {"connected":true,"peerAddress":"192.168.0.54","socketId":2,"name":"","bufferSize":4096,"peerPort":502,"localAddress":"192.168.0.56","persistent":false,"localPort":49507,"paused":false}
]
```

I can disconnect/close the socket 1 (already not connected):

```
chrome.sockets.tcp.disconnect(1, function() { console.log('ok')})
ok
chrome.sockets.tcp.close(1, function() { console.log('ok')})
ok
```

But I can not disconnect from and active socket (ID 2 in the example):

```
chrome.sockets.tcp.disconnect(2, function() { console.log('ok')})
(NO RESPONSE / HANGING)
```

Same for:

```
chrome.sockets.tcp.close(2, function() { console.log('ok')})
(NO RESPONSE / HANGING)
```

Note: if I lock the screen and resume the application, all sockets are disconnected by the system.

**Please tell me if you need more information** (I am newbie in the cordova world)

Davide

**Comment by KoenLav** [comment]
Same problem here!

**Comment by KoenLav** [comment]
I've filed a pull request which addresses this bug.

**Comment by agrieve** [comment]
Published plugin with this patch as v1.3.7
